src/docfx/build/Build.cs

```diff
@@ -12,19 +12,20 @@ namespace Microsoft.Docs.Build
 {
     internal static class Build
     {
-        public static Task Run(string workingDirectory, CommandLineOptions options)
+        public static async Task<int> Run(string workingDirectory, CommandLineOptions options)
```

Comment:
consider use `bool` here to represent `hasError`
```suggestion
        public static async Task<bool> Run(string workingDirectory, CommandLineOptions options)
```

Reply:
The exit code can be used directly for the caller, will update the inner method to return `Task<bool>` to identify execution succeeded.